n_utils/tests/profile_util_test.py

```diff
@@ -31,7 +31,7 @@ def test_adfs_profile_switch(capsys):
     assert "adfs-aws-login --profile testprofile" in captured.out
 
 
-def test_adfs_profile_switch(capsys):
+def test_lastpass_profile_switch(capsys):
     enable_profile("lastpass", "testprofile")
     captured = capsys.readouterr()
-    assert "lastpass-aws-login --profile testprofile"
+    assert "lastpass-aws-login --profile testprofile" in captured.out
```

Comment:
This test was not doing anything and the name conflicted with the one above 🤔 Is this needed or should it be removed?

Reply:
Not a huge test, but supposed to test enabling lastpass saml authenticated profiles. I guess that's a cut & paste bug (incomplete and forgot to change function name).